include/odp/api/spec/timer.h

```diff
@@ -135,6 +135,21 @@ typedef struct {
  * Timer capability
  */
 typedef struct {
+	/** Common for all clock sources */
+	struct {
+		/** Maximum number of timer pools of any clock source */
+		unsigned int max_pools;
+	} common;
+
+	/** Maximum number of timer pools */
+	unsigned int max_pools;
```

Comment:
Barry pointed out at today's ODP public call that these `unsigned int` fields should be `uint32_t` for consistency with standard usage in ODP APIs.

Reply:
Makes sense. As a separate change we might want to clean-up couple other structs which use `unsigned` (`odp_pool_capability_t.max_pools`, `odp_pktio_capability_t.max_input_queues`, `odp_queue_capability_t.max_sched_groups` etc.).